packages/nextjs/pages/projects/[projectId].tsx

```diff
@@ -1,88 +1,147 @@
 import React, { useEffect, useState } from "react";
-import type { NextPage } from "next";
+import { NextPage } from "next";
 import { GetServerSideProps } from "next";
-import { HeartIcon as HeartFilledIcon } from "@heroicons/react/20/solid";
-import { HeartIcon } from "@heroicons/react/24/outline";
-import { Address } from "~~/components/scaffold-eth";
-import SuggestProjects from "~~/components/shared/SuggestProjects";
-import { useBallot } from "~~/context/BallotContext";
+import { AiOutlineGithub } from "react-icons/ai";
+import { BsGlobe } from "react-icons/bs";
+import { IoMdOpen } from "react-icons/io";
+import { useAccount } from "wagmi";
+import { ArrowTopRightOnSquareIcon } from "@heroicons/react/20/solid";
+import YourBallot from "~~/components/op/projects/YourBallot";
+import IncludedProjects from "~~/components/project/IncludedProjects";
+import ProjectHeader from "~~/components/project/ProjectHeader";
+import Sidebar from "~~/components/shared/Sidebar";
 import dbConnect from "~~/lib/dbConnect";
 import Project, { ProjectDocument } from "~~/models/Project";
-import { notification } from "~~/utils/scaffold-eth";
 
 interface Props {
   projects: ProjectDocument[];
 }
 
 const ProjectDetail: NextPage<Props> = ({ projects }) => {
-  const [isLiked, setIsLiked] = React.useState(false);
-  const [isAdded, setIsAdded] = useState(false);
-  const { state, dispatch } = useBallot();
+  const { isDisconnected } = useAccount();
+  const [wallet, setWallet] = useState<boolean | false>(false);
+  // TODO: Add these entries to the database for each document and pull from there
+  const impactArray = [
```

Comment:
Can you please shed some light on what this "Impact array" is?

Reply:
We circled up in the channel about this. Marking as resolved.